app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,11 @@
 class PostsController < ApplicationController
   def index
-    @user = current_user
+    @user = User.find(params[:user_id])
     @posts = @user.posts.paginate(page: params[:page], per_page: 4).includes(:comments, :likes)
```

Comment:
## Optional Change 

- [ ] I highly recommend you add `load_and_authorize_resource` in this file for authorization setup. It's required but the previous review did not pick that up. Kindly consider adding that code at the top of this class 👍🏽

Reply:
okay, thank you for the advise!